yarn.lock

```diff
@@ -5,7 +5,6 @@
 "101@^1.0.0", "101@^1.2.0":
   version "1.6.3"
   resolved "https://registry.yarnpkg.com/101/-/101-1.6.3.tgz#9071196e60c47e4ce327075cf49c0ad79bd822fd"
-  integrity sha512-4dmQ45yY0Dx24Qxp+zAsNLlMF6tteCyfVzgbulvSyC7tCyd3V8sW76sS0tHq8NpcbXfWTKasfyfzU1Kd86oKzw==
```

Comment:
not sure what happened here, these hashes should not disappear. @tjshipe did you add them with a special command?